### ISSUE TITLE
Could you update es_freeroam

### ISSUE THREAD

**Issue by keketiger** [Lokiro67/FiveM-Script#1]
With the new version of essentialmode "essentialmode 5" it is impossible to use es_freeroam !

**Comment by tomames360** [comment]
nah, gotta update it yourself or asking very nicely to update it

**Comment by ghermans** [comment]
This is actually a old fork -> you can find the new "freeroam" here:
https://github.com/FiveM-Scripts/fs_freemode
